### DISCUSSION TITLE
Channels and categories are hard to differ

### DISCUSSION THREAD

**Discussion by demirdegerli** [stoatchat/discussions#564] [Feature Suggestions]
Categories must have a smaller text size than channels (i know they are but it's unnoticeable) and less opacity (more gray).

# Discord'sㅤ UIㅤ ㅤ ㅤ ㅤ ㅤ ㅤ ㅤ ㅤ ㅤ ㅤ ㅤ ㅤ ㅤ Revolt's UI
![image](https://github.com/revoltchat/revolt/assets/67432156/caf5a0c6-9e98-48c9-a566-fab7a8ed85e1) ![image](https://github.com/revoltchat/revolt/assets/67432156/f5493b75-7744-4668-9584-c29fdcc52b28)

**Comment by DragShot** [comment]
Personally, I find that removing the bold style from the channel names gets rid of the problem for me, as it makes it much easier to tell categories and channels apart.
